App.py

```diff
@@ -358,6 +360,12 @@ def create_group(person):
     person.date.updated = datetime.datetime.utcnow()
     person.save()
 
+    if invite is not None:
+        if type(invite) is not array:
+            return jsonify({'msg': 'Missing Required Field(s) / Invalid Type(s).'}), 400
+        for email in invite:
+            group.invite.append(email);
+
     # TODO - finish this when invites are implemented
     if members is not None:
         if type(members) is not array:
```

Comment:
this is no longer needed because you are checking for invites instead of members